packages/synapse-react-client/src/components/QueryContext/QueryContext.tsx

```diff
@@ -9,16 +9,6 @@ import {
 import { ImmutableTableQueryResult } from '../../utils/hooks/useImmutableTableQuery/useImmutableTableQuery'
 import { ReadonlyDeep } from 'type-fest'
 
-/*
-  For details page: to lock a column (e.g. study, grant) so that the facet values and active filters
-  will not appear on the details page. The facet name is given by the URL search param.
-  The type is defined here so that other child components in SRC won't generate type errors.
- */
-export type LockedColumn = {
-  columnName?: string
-  value?: string
-}
```

Comment:
In this PR I updated queryUtils.ts but I noticed that `queryUtils` imported this `LockedColumn` type. To prevent circular dependencies, `utils/` should never import from `components/`. So I moved `LockedColumn` to `utils/types`